src/main/java/com/js/tictactoe/parser/DigitParser.java

```diff
@@ -14,19 +14,25 @@ public static String correctCoordinates(Supplier<String> input, Consumer<String>
     do {
       line = input.get();
 
-      if (line.equalsIgnoreCase("quit"))
-        return line;
+      if (returnQuitLine(line)) return line;
 
       isNumber = DigitParser.isInputContainingDigits(line);
-
-      if (!isNumber)
-        output.accept(reader.loadString("coords"));
+      showIncorrectNumberMessage(output, reader, isNumber);
 
     } while (!isNumber);
     return line;
   }
 
+  private static boolean returnQuitLine(String line) {
```

Comment:
gdzieś to widziałem:

+  private boolean isExit(String line) {
+    return line.equalsIgnoreCase("quit");
+  }